build.gradle

```diff
@@ -20,62 +16,49 @@ tag {
     }
 }
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+subprojects {
+    apply plugin: 'java'
+    apply plugin: 'idea'
+    apply plugin: 'jacoco'
+    apply plugin: "com.srcclr.gradle"
```

Comment:
I don't see the java, idea, and jacoco plugins defined in the subprojects, are they loaded by default?

Reply:
Try running
`./gradlew gradle-plugin:publishToMavenLocal`

Reply:
When I run `./gradlew gradle-plugin:publishToMavenLocal`                          

I get this error:
```                                                          
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=512m; support was removed in 8.0
Listening for transport dt_socket at address: 5005

FAILURE: Build failed with an exception.

* What went wrong:
Task 'publishToMavenLocal' not found in project ':gradle-plugin'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

Reply:
Not working for me too

Reply:
Fixed